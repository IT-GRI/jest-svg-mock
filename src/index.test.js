const {it, expect} = require('@jest/globals')

const svg= require('../fixtures/test.svg');


it('svg should match snapshot', () => {
    expect(svg).toMatchSnapshot();
});
