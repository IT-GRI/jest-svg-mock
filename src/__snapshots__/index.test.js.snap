// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`svg should match snapshot 1`] = `"<svg data-file-name='svg-test'></svg>"`;
